src/content/get-started/install/amazon-eks.mdx

```diff
@@ -297,6 +297,72 @@ okteto-ingress-nginx-controller  LoadBalancer   10.0.7.73    a519c8b3b27f94...el
 
 You'll need to take the `EXTERNAL-IP` address (`a519c8b3b27f94...elb.amazonaws.com` in the example), and add it to your DNS for the domain you have chosen to use. In Route 53, this is done by creating an `A` record with the name  `*`, pointing to the alias of the Elastic Load Balancer.
 
+### Configure a private registry with Amazon ECR
```

Comment:
I think this creates additional friction when installing Okteto in AWS. Configuring ECR access is optional.

What I would have is a dedicate page in the `Administration` menu to talk about private registry configuration, and then, a section for ECR, Dockerhub, Google, Azure, ...